pkg/scheduler/plugins/gang/gang_cache.go

```diff
@@ -0,0 +1,198 @@
+package gang
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/scheduler-plugins/apis/scheduling/v1alpha1"
+
+	"github.com/koordinator-sh/koordinator/apis/extension"
+	schedulingconfig "github.com/koordinator-sh/koordinator/apis/scheduling/config"
+)
+
+type gangCache struct {
+	lock       *sync.RWMutex
+	gangItems  map[string]*Gang
+	pluginArgs *schedulingconfig.GangSchedulingArgs
+}
+
+func NewGangCache() *gangCache {
```

Comment:
please add pluginArgs to NewGangCache's parameter

Reply:
done